source/deployment/bulk-loading-data-format.rst

```diff
@@ -8,8 +8,12 @@ The input data file must be a valid `JSONL
 
 Version
   Mandatory. The Version object must be the first line in the file, and must occur only once.
+Scheme
+  Optional. If present, Scheme objects must occur after the Version object but before any Team objects.
+Emoji
+  Optional. If present, Emoji objects must occur after the Version objects but before any Team objects.
```

Comment:
Based on order of objects: 
- Does Scheme need to occur after Version object before Emoji objects?
- Does Emoji have to occur after the Scheme object prior to Team?

Reply:
No to both, but if it makes it simpler to explain to users, we could just say it does in the docs. There's no real downside of forcing that strict ordering.

Reply:
I was just making sure there was not a hard requirement and an oversight possibly in copy/paste.  Enforcing the order in documentation may make it more consistent with the other items and avoid the question that I asked.